server/backend/testspg/router/test_router_user_role_assignment.py

```diff
@@ -428,103 +423,312 @@ def test_user_role_forest_client_number_not_exist_bad_request(
-        "forest_client_number": client_number_not_exists
+        "forest_client_number": client_number_not_exists,
     }
     response = test_client_fixture.post(
         f"{endPoint}",
         json=invalid_request,
-        headers=jwt_utils.headers(fom_dev_access_admin_token)
+        headers=jwt_utils.headers(fom_dev_access_admin_token),
     )
     assert response.status_code == HTTPStatus.BAD_REQUEST
     assert response.json() is not None
-    assert (f"Forest Client Number {client_number_not_exists} does not exist."
-            in response.json()["detail"]
-            )
+    assert (
+        f"Forest Client Number {client_number_not_exists} does not exist."
+        in response.json()["detail"]
+    )
 
 
 def test_user_role_forest_client_number_inactive_bad_request(
-    test_client_fixture: TestClient,
-    fom_dev_access_admin_token
+    test_client_fixture: TestClient, fom_dev_access_admin_token
 ):
     """
     Test assign user role with inactive forest client number should be
     rejected.
     """
     invalid_request = {
         **TEST_USER_ROLE_ASSIGNMENT_FOM_DEV_ABSTRACT,
-        "forest_client_number": CLIENT_NUMBER_EXISTS_DEACTIVATED
+        "forest_client_number": CLIENT_NUMBER_EXISTS_DEACTIVATED,
     }
     response = test_client_fixture.post(
         f"{endPoint}",
         json=invalid_request,
-        headers=jwt_utils.headers(fom_dev_access_admin_token)
+        headers=jwt_utils.headers(fom_dev_access_admin_token),
     )
     assert response.status_code == HTTPStatus.BAD_REQUEST
     assert response.json() is not None
-    assert ("Forest Client is not in Active status")
+    assert "Forest Client is not in Active status"
 
 
 def test_self_grant_fail(
     test_client_fixture: starlette.testclient.TestClient,
     fom_dev_access_admin_token,
-    db_pg_session: Session
+    db_pg_session: Session,
 ):
     # Setup challenge: The user in the json sent to the service must match the user
     # in the JWT security token.
     user_role_assignment_request_data = {
         "user_name": jwt_utils.IDIR_USERNAME,
         "user_type_code": UserType.IDIR,
-        "role_id": TEST_FOM_DEV_REVIEWER_ROLE_ID
+        "role_id": TEST_FOM_DEV_REVIEWER_ROLE_ID,
     }
 
     response = test_client_fixture.post(
         f"{endPoint}",
         json=user_role_assignment_request_data,
-        headers=jwt_utils.headers(fom_dev_access_admin_token)
+        headers=jwt_utils.headers(fom_dev_access_admin_token),
     )
 
     row = db_test_utils.get_user_role_by_cognito_user_id_and_role_id(
-        db_pg_session,
-        jwt_utils.COGNITO_USERNAME,
-        TEST_FOM_DEV_REVIEWER_ROLE_ID
+        db_pg_session, jwt_utils.COGNITO_USERNAME, TEST_FOM_DEV_REVIEWER_ROLE_ID
     )
     assert row is None, "Expected user role assignment not to be created"
 
     jwt_utils.assert_error_response(response, 403, ERROR_SELF_GRANT_PROHIBITED)
 
 
+# ---------------- Test delete user role assignment ------------ #
```

Comment:
the changes above this line in this file are all from auto formatting, new tests are added below